File: src/main/java/EbbinghausReminderRunner.java
public class EbbinghausReminderRunner {

    public static void main(String[] args) {
        EbbinghausReminder reminder = new EbbinghausReminder("2014-12-12");
        for (String date : reminder.datesToBeReviewed()) {
            System.out.println(date);
        }
        System.out.println("The old ones:");
        System.out.println(reminder.randomDatesToBeReviewedAfter("2014-08-22"));
    }

}

File: ebbinghaus_reminder_runner.rb
require_relative 'lib/ebbinghaus_reminder'

if ARGV.size == 0
  reminder = EbbinghausReminder.today()
else
  reminder = EbbinghausReminder.today_is(ARGV[0])
end

for date in reminder.dates_to_be_reviewed() do
  puts date
end

for date in reminder.random_dates_to_be_reviewed_after("2014-08-22") do
  puts date
end